tiered-storage/pom.xml

```diff
@@ -32,6 +32,11 @@
   <artifactId>tiered-storage-parent</artifactId>
   <name>Apache Pulsar :: Tiered Storage :: Parent</name>
 
+  <properties>
+    <!-- pin the jclouds-shaded version to make the pulsar build friendly to intellij -->
+    <pulsar.jclouds.shaded.version>2.3.0</pulsar.jclouds.shaded.version>
```

Comment:
The current version in master branch is actually 2.4.0-SNAPSHOT. Also we would have to make sure this is updated at each release

Reply:
> The current version in master branch is actually 2.4.0-SNAPSHOT. Also we would have to make sure this is updated at each release

The latest version maven repo is 2.3.0, so I forced the version to be specified, like [protobuf_shaded](https://github.com/apache/pulsar/blob/master/pom.xml#L143). 
This can lead to inflexibility, If build successfully with project.version IDEA, still did not introduce the correct jar. 
Do you have a better way of handling it? thanks.

Reply:
Got it.. I was mistakenly thinking this was on the "real" code :)